packages/components/src/navigator/types.ts

```diff
@@ -16,8 +22,13 @@ export type NavigatorLocation = NavigateOptions & {
 
 export type NavigatorContext = {
 	location: NavigatorLocation;
-	goTo: ( path: string, options?: NavigateOptions ) => void;
+	goTo: (
+		path: string,
+		animationOverride?: AnimationOverride,
```

Comment:
Should we make this the third argument, so we don't have to modify the existing calls to it?